SConstruct

```diff
@@ -88,6 +89,9 @@ env_base.__class__.disable_warnings = methods.disable_warnings
 env_base["x86_libtheora_opt_gcc"] = False
 env_base["x86_libtheora_opt_vc"] = False
 
+# avoid issues when building with different versions of python out of the same directory
+env_base.SConsignFile(".sconsign{0}.dblite".format(pickle.HIGHEST_PROTOCOL))
```

Comment:
This will appear as untracked files since it no longer matches the line in `.gitignore`.